src/main/java/io/confluent/kafkarest/ProducerPool.java

```diff
@@ -143,7 +144,8 @@ private static String getBootstrapBrokers(ZkUtils zkUtils) {
         if (bootstrapBrokers.length() > 0) {
           bootstrapBrokers += ",";
         }
-        bootstrapBrokers += ep.connectionString();
+        String hostport = ep.host() == null ? ":" + ep.port() : Utils.formatAddress(ep.host(), ep.port());
+        bootstrapBrokers += ep.securityProtocol() + "://" + hostport;
```

Comment:
Ooops.  Looking more closely, it accepts backslashes but not forward slashes.  Go figure.